.gitignore

```diff
@@ -32,6 +32,7 @@ bld/
 temp/
 _common/
 _build/
```

Comment:
I think in this PR, we should also delete the generated buildconfigs in _generated

Reply:
Are generated buildConfigs referring to versionmap.txt files, or other files?

Reply:
The versionmap.txt files must continue to be in source control.  The subdirectories under _generated are to be deleted